.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_SERVER_URL=http://localhost:8001
```

Comment:
I believe this should go into .env.development and there should be a corresponding .env.production